titan/output.py

```diff
@@ -132,14 +135,14 @@ def get_stats(
     args:
         all_agents: all of the agents in the population
         new_hiv.dx: agents who are newly diagnosed with hiv this timestep
-        deaths: agents who died this timestep
+        exits: dictionary indicated how many agents exited by exit type this timestep
```

Comment:
Is this exit type or class? I think it should be class as there could be multiple classes using the same type that we want to track separately